src/GovUk.Education.ExploreEducationStatistics.Data.Services/MetaGuidanceSubjectService.cs

```diff
@@ -139,11 +154,11 @@ private async Task<MetaGuidanceSubjectViewModel> BuildSubjectViewModel(ReleaseSu
                 Id = subject.Id,
                 Content = releaseSubject.MetaGuidance ?? "",
                 Filename = subject.Filename ?? "Unknown",
```

Comment:
Now we have the content database connection and you have added that as a dependency here and fetched the ReleaseFile, we don't need to access `subject.filename`, which means we no longer need to add the filename to Subject when creating a subject and synchronise them across Public/Private with DF.

Reply:
I've changed the bit you refer to, but I think I'll create a separate ticket to remove Filename from Subject